.circleci/config.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       - run:
           name:  Run E2E test
           command: go test -tags=e2e -parallel 24 -v -timeout 20m  ./test/e2e/... -identifier=$CIRCLE_BUILD_NUM
-          no_output_timeout: 15m
+          no_output_timeout: 25m
```

Comment:
We can get the logs by using kubectl, its just not implemented yet.

Also this config option is not a timeout but a timeout if there is no output. IMHO we should leave it as is, usually we have some output in between because of alle pause/continue or because the tests on a cloud provider finished

Reply:
right, if this timeout is lower that the total time needed to run the test the process will be killed and we won't get any logs. Here is the output

```
--- PASS: TestHetznerProvisioningE2E (0.00s)
    --- PASS: TestHetznerProvisioningE2E/scenario_3_Ubuntu_CRI-O_1.9 (136.54s)
    --- PASS: TestHetznerProvisioningE2E/scenario_1_Ubuntu_Docker_1.13 (151.55s)
    --- PASS: TestHetznerProvisioningE2E/scenario_2_Ubuntu_Docker_17.03 (166.53s)
    --- PASS: TestHetznerProvisioningE2E/scenario_4_CentOS_Docker_1.12 (211.53s)
    --- PASS: TestHetznerProvisioningE2E/scenario_5_CentOS_Docker_1.13 (271.43s)
Too long with no output (exceeded 15m0s)

```

Reply:
Ouh okay, interesting, I haven't seen us running into that one before.

I'd not alter the timeout in the verify thought, I think the timeout on the test command  itself should be our only one.

Reply:
> I'd not alter the timeout in the verify thought, I think the timeout on the test command itself should be our only one.

If this timeout is bigger, then there is no way to exit the process in a graceful way, it will be killed by the test framework. It might also have negative impact on potential troubleshooting as I'm not sure if we will observe failing due to timeouts in the logs.

Reply:
Yeah but the timeout in the test framework is supposed to be the only one.

If that happens the e2e test will fail. Its a pretty obvious error as there is no other case where the test framework kills the test.

I simply want to avoid having more than one know to adjust the timeout.